### PR TITLE
Fix SCEta locally

### PR DESCRIPTION
**Pull request by steeevin88**
- currently, running "docker-compose -f docker-compose.dev.yml up" leads to an NGINX error when accessing the frontend
- the frontend's current Dockerfile.dev isn't needed anymore since everything goes through NGINX
- readded Prometheus container on dev to test current+future metrics

### CODE REVIEW

**Review by evanugarte — COMMENTED** [review]
the dockerfile.dev allows the sceta frontend to reload locally

File: frontend/docker/Dockerfile.dev [hunk 1]
```diff
@@ -15,4 +15,4 @@ COPY tailwind.config.js ./tailwind.config.js
 
 RUN npx tailwindcss build -i src/input.css -o public/output.css --minify
 
-CMD ["npm", "start"]
+CMD ["npm", "start"]
```
Comment (evanugarte) [inline frontend/docker/Dockerfile.dev]: undo the changes to this file

File: docker-compose.dev.yml [hunk 2]
```diff
@@ -28,9 +28,18 @@ services:
...
             - WATCHFILES_FORCE_POLLING=true
+    prometheus:
+        image: prom/prometheus:latest
+        restart: always
+        volumes:
+            - ./prometheus.yml:/etc/prometheus/prometheus.yml
+        command:
+            - --config.file=/etc/prometheus/prometheus.yml
+        ports:
+            - 9090:9090
```
Comment (evanugarte) [inline docker-compose.dev.yml]: put this behind nginx, at `/prometheus`, we can do this in the nginx.dev.conf like
```nginx
       location /prometheus/ {
            resolver 127.0.0.11 valid=15s;

            proxy_set_header   Host $host;
            set $upstream http://prometheus:9090;
            proxy_pass $upstream;

            rewrite ^/prometheus/?$ /prometheus/graph redirect;
            rewrite ^/prometheus(.*)$ $1 break;
        }
```

File: frontend/docker/Dockerfile.dev (hunk 1 above)
Comment (steeevin88) [reply frontend/docker/Dockerfile.dev]: fixed

File: docker-compose.dev.yml (hunk 2 above)
Comment (steeevin88) [reply docker-compose.dev.yml]:
```
rewrite ^/prometheus/?$ /prometheus/graph redirect;
```
I tried adding this line but it routes me to port 80 instead of port 3001

```
rewrite ^/prometheus/?$ http://$host:3001/prometheus/graph redirect;
```
this worked as `localhost:3001/prometheus/` would go to `localhost:3001/prometheus/graph` as intended but `localhost:3001/prometheus` goes to `localhost/prometheus/graph` again

should i still add this?

**Review by evanugarte — APPROVED** [review]
very very nice job

File: docker-compose.dev.yml (hunk 2 above)
Comment (steeevin88) [reply docker-compose.dev.yml]: resolved
